packages/kernel-6.1/config-bottlerocket

```diff
@@ -116,6 +119,12 @@ CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
 # Disable SafeSetID LSM
 # CONFIG_SECURITY_SAFESETID is not set
 
+# Enable landlock
+CONFIG_SECURITY_LANDLOCK=y
+
+# Add landlock to the list of LSMs.
+CONFIG_LSM="landlock,lockdown,yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor,bpf"
```

Comment:
This is the upstream default for kernel 6.1: [Kconfig](https://elixir.bootlin.com/linux/v6.1.146/source/security/Kconfig#L285).